acme/dns_challenge.go

```diff
@@ -124,19 +202,20 @@ func unFqdn(name string) string {
 	return name
 }
 
-// waitFor polls the given function 'f', once per second, up to 'timeout' seconds.
-func waitFor(timeout int, f func() (bool, error)) error {
+// waitFor polls the given function 'f', once every 'interval' seconds, up to 'timeout' seconds.
+func waitFor(timeout, interval int, f func() (bool, error)) error {
```

Comment:
That was on my mind too! :smile: But maybe we can do the code re-organization later in a separate patch?